src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml

```diff
@@ -39,6 +39,20 @@
             <argument type="service" id="session" />
         </service>
 
+        <service id="Sylius\Bundle\AdminBundle\Action\Account\GetRequestPasswordResetAction" public="true">
+            <argument type="service" id="twig"/>
+            <argument type="service" id="form.factory"/>
+        </service>
+
+        <service id="Sylius\Bundle\AdminBundle\Action\Account\PostRequestPasswordResetAction" public="true">
+            <argument type="service" id="form.factory"/>
+            <argument type="service" id="sylius.command_bus"/>
+            <argument type="service" id="session.flash_bag"/>
+            <argument type="service" id="router"/>
+
+            <tag name="controller.service_arguments" />
```

Comment:
why do we need this tag, when service is already public?

Reply:
@AdamKasp As I remember it passes `Request` obj as an argument to `__invoke()` method. I'll check if I'm right, if it turns out it's redundant I'll remove it.

Reply:
I was wrong, I'll remove it :D.